README.md

```diff
@@ -64,7 +75,7 @@ To get started with Nesis
       6. Username: **your_username**
       7. Password: **your_password**
       8. Click **Create**
-9. After about 5 minutes, the background process will start indexing documents and then you should be able to query your documents.
+      9. After about 5 minutes, the background process will start indexing your documents and then you should be able to converse with your documents.
```

Comment:
@namwanza this step should be re-worded. #27 enables users to run datasource ingestion tasks on the fly. So we need to state something like
> Run an adhoc ingestion by clicking **_Ingest_** but of the datasource.

Reply:
@mawandm, Okay, I'll make the changes and create another pull request.